var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -39,9 +39,6 @@ class Fftw(Package):
     version('3.3.5', '6cc08a3b9c7ee06fdd5b9eb02e06f569')
     version('3.3.4', '2edab8c06b24feeb3b82bbb3ebf3e7b3')
 
-    patch('pfft-3.3.5.patch', when="@3.3.5+pfft_patches", level=0)
```

Comment:
not sure what's the best way to rebase, maybe try merging the upstream `develop`?